data/mods/ssb/statuses.js

```diff
@@ -1595,6 +1607,22 @@ let BattleStatuses = {
 			this.add('-end', pokemon, 'Attract', '[silent]');
 		},
 	},
+	// Custom effect for Rage's multihit
+	hitsmorethanonce: {
```

Comment:
Maybe call it `enrageeeeed` because I feel that fits better.

Reply:
._.

Reply:
:3